schedule/yast/ssh-x.yaml

```diff
@@ -6,10 +6,12 @@ vars:
   DESKTOP: textmode
   PATTERNS: minimal,base
   VIDEOMODE: ssh-x
+  YUI_REST_API: 1
```

Comment:
this one in theory should not work either, according to some of my recent findings in other task.